frontend/src/views/account/Profile.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Text, MediumText, LargeText } from '../../components/core/Text';
+import { PrimaryButtonLink } from '../../components/core/Link';
+import { useAppSelector } from '../../util/Hook';
+
+const Content = styled.div`
+  text-align: center;
+`;
+
+function DashboardPage() {
```

Comment:
Did you mean to make this page publicly-accessible via URL (`/profile`) without logging in? Wouldn't we want only logged-in users to access it because there is no information otherwise?

Reply:
Nope, good catch - changed this to use a PrivateRoute